code/modules/networks/computer3/mainframe2/misc_terms.dm

```diff
@@ -3838,6 +3838,12 @@ TYPEINFO(/obj/machinery/networked/test_apparatus)
 			boutput(user, "That is far too big to fit!")
 			return
 
+	MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
+		if (!istype(O,/obj/) || O.anchored) return
+		if (BOUNDS_DIST(src, O) > 0 || !isturf(O.loc)) return
+		if (!in_interact_range(user, O) || !in_interact_range(user, src) || !isalive(user)) return
+		src.attackby(O, user)
```

Comment:
```suggestion
		src.Attackby(O, user)
```
Always override `attackby` and call `Attackby`, the latter has some signal-sending stuff that we pretty much always want to happen